From da8227c4a4b1f3d02ce7ea0e5e2f00e8d3c71950 Mon Sep 17 00:00:00 2001
From: Apekshit <apeksharma@gmail.com>
Date: Mon, 11 Jan 2016 12:04:53 -0800
Subject: [PATCH 209/217] HBASE-14159 Add dependency on
 hadoop-mapreduce-client-jobclient from hbase-spark.

fix hbase-spark pom to include hadoop-mapreduce-client-jobclient test-jar

Adds a test-scope dependency to get rid of following error:

[WARNING] warning: Class org.apache.hadoop.mapred.MiniMRCluster not found - continuing with a stub.
[WARNING] one warning found

Change-Id: Ia05228fd2bb2ff6ec3c956472f39b0f68c6bf90f
Signed-off-by: Sean Busbey <busbey@cloudera.com>
Reason: New Feature
Author: Apekshit Sharma
Ref: CDH-35578
---
 hbase-spark/pom.xml |   13 ++++++++++---
 1 file changed, 10 insertions(+), 3 deletions(-)

diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index 3888fed..449a3f6 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -520,6 +520,13 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
+            <version>${hadoop-two.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
 
@@ -600,10 +607,10 @@
         </plugins>
     </build>
 
-  <profiles>
+    <profiles>
     <!-- Skip the tests in this module -->
-      <profile>
-        <id>skipSparkTests</id>
+        <profile>
+            <id>skipSparkTests</id>
             <activation>
                 <property>
                     <name>skipSparkTests</name>
-- 
1.7.9.5

